### PR TITLE
lookup: re-add bluebird

### PR DESCRIPTION
**Pull request by alfonsograziano**
##### Checklist

- [x] `npm test` passes
- [x] contribution guidelines followed
      [here](https://github.com/nodejs/citgm/blob/HEAD/CONTRIBUTING.md)

### CODE REVIEW

**Comment by Trott** [comment]
It will take a while for the job to start, but https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/3285/ will hopefully have bluebird results.

**Review by Trott — CHANGES_REQUESTED** [review]
Bluebird passes locally but fails in Jenkins CI.

File: lib/lookup.json [hunk 1]
```diff
@@ -55,6 +55,10 @@
     "prefix": "v",
     "skip": ["win32", "aix"]
   },
+  "bluebird": {
+    "prefix": "v",
+    "maintainers": "petkaantonov"
```
Comment (Trott) [inline lib/lookup.json:60]:
```suggestion
    "maintainers": "petkaantonov",
    "stripAnsi": true
```

**Comment by Trott** [comment]
Running again with ANSI stipping enabled, just in case that's somehow the source of the problem. https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/3286/

**Comment by alfonsograziano** [comment]
Still failing, to be honest I'm not sure if it is a legittimate issue or a false positive

**Comment by Trott** [comment]
The ANSI-stripping didn't do anything either. Harumph.

> (I mean when locally it works but fails on the CI pipeline)

I wish I had an answer. I have no idea why this passes locally and fails *everywhere* in CI. Must be something in the CI configuration somewhere, but I'm not even sure what to look for. Suggestions welcome.....

@nodejs/build 

TBH, unless bluebird is important to you personally, I'm not sure this is worth a lot of effort. Bluebird hasn't seen a new version if 4 years and is likely mostly obsolete at this point. Its current download stats are an artifact of its enormous popularity and utility in the past.

**Comment by alfonsograziano** [comment]
Good point, probably is not worth to add this package. Is definitely weird that the pipeline is failing on the CI. If the same happens with other packages, then probably is worth to investigate. 

I'll close this PR for now and I'll focus on other packages that we should add due to their popularity in the ecosystem. 

Thanks for your support, really appreciated 😃

**Pull request closed without merging**